src/proto/faabric.proto

```diff
@@ -114,7 +108,6 @@ message Message {
     int32 returnValue = 11;
 
     string snapshotKey = 12;
-    int32 snapshotSize = 13;
```

Comment:
Neither this nor the `snapshotSize` variable above were actually used. I've deleted them but not updated the numbers of the subsequent fields to save on the diff size.